src/admin-views/tpp-metabox-sku.php

```diff
@@ -27,6 +27,10 @@ class="ticket_field sku_input ticket_form_right"
 		value="<?php echo esc_attr( $sku ); ?>"
 	>
 	<p class="description ticket_form_right">
-		<?php esc_html_e( 'A unique identifying code for each ticket type you\'re selling', 'event-tickets' ); ?>
+		<?php echo esc_html( sprintf(
```

Comment:
I could not find a note in the standards, but check the coding and where you have a opening <?php 
Lets be consistent and  start the code on the next line or right after. 
```
<?php
echo esc_html(
)
?>
```
or

```
<?php echo esc_html(
)
?>
```

Inline PHP would remain the same.

Reply:
done